source/agora/consensus/Genesis.d

```diff
@@ -94,6 +95,68 @@ private immutable Block UnitTestGenesisBlock = {
         prev_block:  Hash.init,
         height:      Height(0),
         merkle_root: UnitTestGenesisMerkleTree[$ - 1],
+        enrollments: [
```

Comment:
So there are 6 enrollments in this genesis block?

Reply:
Yep